### PR TITLE
Team Management

### PR DESCRIPTION
**Pull request by nam-t24**
Run `npm install` again for the team member dropdown menu
### Team Management
Owner:
- transfer ownership

Admin:
- demote admins
- make admin
- kick any user except owner

Member:
- view profile
- copy email

Higher ranks have all functionalities of lower ranks
Screenshot 2024-02-29 at 1 49 19 PM

### Tier Functionalities
Owner and admins can edit org info and invite users
Screenshot 2024-02-29 at 1 52 14 PM
Error handling is implemented. If user email doesn't exist or if user is already in org, correct error toast will pop up.

### Leaving Org Restrictions
Added restrictions to leaving orgs

Owner must transfer ownership before leaving an org. If owner is the only person in the org, they can leave without transferring ownership

### CODE REVIEW

File: app/dashboard/organization/overview/page.tsx [hunk 1]
```diff
@@ -20,17 +21,19 @@ import {
...
+    isNonProfit: boolean;
+    owner: string;
+    admins: Array<string>;
 }
 
 export default function OrganizationOverview() {
     const router = useRouter();
     const { toast } = useToast();
-    const [userID, setUserID] = useState("");
+    const [userID, setUserID] = useState(null);
```
Comment (Andy581) [inline app/dashboard/organization/overview/page.tsx]: useStates that have a null value initialized should have a datatype specified for better readability

File: app/dashboard/organization/overview/page.tsx [hunk 2]
```diff
@@ -41,7 +44,8 @@ export default function OrganizationOverview() {
     const [userOrgID, setUserOrgID] = useState(-1);
     const [loading, setLoading] = useState(true);
     const [isOwner, setIsOwner] = useState(null);
```
Comment (Andy581) [inline app/dashboard/organization/overview/page.tsx]: For some reason I can't highlight multiple lines and group them as a single comment, so same comment applies here

File: utils/scripts/team.ts [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        console.log(errorAdmin.message);
+        return [];
+    }
+    const admins = adminData[0].admins;
+
+    const { data, error } = await supabase.from('profiles').select('id, full_name, email').eq('FK_organizations', orgID);
+    if(error){
+        console.log(error.message);
+        return [];
+    }
```
Comment (Andy581) [inline utils/scripts/team.ts]: How will we distinguish whether a returned empty array means there was an error on the server side or there is just no data for the array to be filled with? We should be either be throwing an error message or null for the client to catch and handle it.

**Review by Andy581 — CHANGES_REQUESTED** [review]

File: utils/scripts/team.ts (hunk 3 above)
Comment (nam-t24) [reply utils/scripts/team.ts]: Whoops this function is unused and didn't flush it out since i didn't end up using it

**Review by Andy581 — APPROVED** [review]

**Pull request merged**